eng/pipelines/ci.yml

```diff
@@ -11,11 +11,11 @@ trigger:
 
 # Run a nightly (PST), to create the alpha release
 schedules:
-  - cron: '0 8 * * *'
+  - cron: '*/20 * * * *' # '0 8 * * *'
     displayName: Nightly Alpha Release Build
     branches:
       include:
-      - main
+      - emlyn/*
```

Comment:
Just checking that this line is expected

Reply:
It was; part of testing.